plugins/modules/info.py

```diff
@@ -125,14 +132,19 @@
     - The list of user mapping rules of PowerScale cluster.
     - The list of ldap providers of PowerScale cluster.
     - SMB global settings - C(smb_global_settings).
-    - NTP servers C(ntp_servers)
-    - Email settings C(email_settings)
-    - Cluster identity C(cluster_identity)
-    - Cluster owner C(cluster_owner)
+    - NTP servers C(ntp_servers).
+    - Email settings C(email_settings).
+    - Cluster identity C(cluster_identity).
+    - Cluster owner C(cluster_owner).
     - SNMP settings - C(snmp_settings).
     - Server certificate - C(server_certificate).
-    - roles - C(roles).
+    - Roles - C(roles).
     - Support assist settings- C(support_assist_settings).
+    - Alert settings - C(alert_settings).
```

Comment:
I know we have been adding a subset to the end of the list, however, is there any chance we could add them in a sorted order alphabetically. That way, I could look at the module documentation and easily find whether a given subset if supported or not by the info module.

Reply:
Team will fix these 2 comments in next release sprint.